docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1508,8 +1546,8 @@ Table redirection
 
 .. include:: table-redirection.fragment
 
-The connector supports redirection from Iceberg tables to Hive tables
-with the ``iceberg.hive-catalog-name`` catalog configuration property.
+The connector supports redirection from Iceberg tables to Hive tables with the
+``iceberg.hive-catalog-name`` catalog configuration property.
 
 File formats
```

Comment:
@mosabua https://github.com/trinodb/trino/pull/17510 really needs to be merged. all the file formats stuff is gone to a new page.

Reply:
@m57lyra I was wondering is there supposed to be an Avro section as well? That would be another ticket if so.

Reply:
Maybe .. separate topic for another PR. I know that the metadata files in Iceberg use Avro .. not sure if Avro can also be used for data files. Anyway .. we can sort that out later